lib/xander/application.ex

```diff
@@ -16,7 +16,7 @@ defmodule Xander.Application do
   defp client_opts do
     network = System.get_env("CARDANO_NETWORK", "mainnet") |> String.to_atom()
     path = System.get_env("CARDANO_NODE_PATH", "/tmp/cardano-node.socket")
-    port = System.get_env("CARDANO_NODE_PORT", "0") |> String.to_integer()
```

Comment:
I believe a `0` like this is used to connect to a local socket file, but it looks like the `0` is also provided [here](https://github.com/wowica/xander/blob/main/lib/xander/client_statem.ex#L157).

So, the user still shouldn't need to know about providing `0`, since setting the `socket` tcp type will populate it for them automatically.

I'm just talking my way through this default change; seems like an improvement for the reasons above :+1: